tests/acceptance/conftest.py

```diff
@@ -0,0 +1,262 @@
+        "operational_issues_dir": operational_issues_dir,
+        "datasource_log_dir": datasource_log_dir,
+        "dataset_resource_dir": dataset_resource_dir,
+        "converted_resource_dir": converted_resource_dir,
+        "cache_dir": cache_dir,
+        "custom_temp_dir": custom_temp_dir,
+    }
+
+
+def get_pipeline_csv_data_with_resources_and_endpoints(
+    pipeline: str = "", resource: str = "", endpoint: str = ""
+):
+    return {
+        "dataset": [pipeline, pipeline, pipeline],
+        "resource": [
+            "",
+            resource,
+            "",
+        ],
+        "endpoint": [
+            "",
+            "",
+            endpoint,
+        ],
+        "column": [
+            "NAME",
+            "res-col-one",
+            "ep-col-one",
+        ],
+        "field": [
+            "name",
+            "res_field_one",
+            "ep_field_one",
+        ],
+    }
+
+
+def copy_latest_specification_files_to(specification_dir: Path):
+    error_msg = "Failed to download specification files"
+    if not specification_dir:
+        pytest.fail(error_msg)
+
+    url_domain = "https://raw.githubusercontent.com"
+    url_path = "/digital-land/specification/main/specification"
+    specification_url = f"{url_domain}{url_path}"
+    specification_csv_list = [
+        "attribution.csv",
+        "collection.csv",
+        "datapackage-dataset.csv",
+        "datapackage.csv",
+        "dataset-field.csv",
+        "dataset-schema.csv",
+        "dataset.csv",
+        "datatype.csv",
+        "field.csv",
+        "issue-type.csv",
+        "licence.csv",
+        "organisation-dataset.csv",
+        "pipeline.csv",
+        "prefix.csv",
+        "project-status.csv",
+        "project.csv",
+        "provision-reason.csv",
+        "schema-field.csv",
+        "schema.csv",
+        "severity.csv",
+        "specification-status.csv",
+        "specification.csv",
+        "theme.csv",
+        "typology.csv",
+    ]
+
+    try:
+        for specification_csv in specification_csv_list:
+            urllib.request.urlretrieve(
+                f"{specification_url}/{specification_csv}",
+                os.path.join(specification_dir, specification_csv),
+            )
+    except URLError:
+        pytest.fail(error_msg)
+
+
+def get_additional_field_csv_data():
+    return {
+        "cardinality": [1, 1],
+        "datatype": ["string", "string"],
+        "description": ["resource field map", "endpoint field map"],
+        "end-date": ["", ""],
+        "entry-date": ["", ""],
+        "field": ["res_field_one", "ep_field_one"],
+        "guidance": ["", ""],
+        "hint": ["", ""],
+        "name": ["res_field_one", "ep_field_one"],
+        "parent-field": ["value", "value"],
+        "replacement-field": ["", ""],
+        "start-date": ["", ""],
+        "text": ["value", "value"],
+        "typology": ["value", "value"],
+        "uri-template": ["", ""],
+        "wikidata-property": ["", ""],
+    }
```

Comment:
do we need this, and any of the related additional field stuff in `test_dirs`? Try remove this function and references to it and see if it still works